cocos/tween/actions/action-interval.ts

```diff
@@ -113,8 +113,8 @@ export abstract class ActionInterval extends FiniteTimeAction {
         return true;
     }
 
-    isDone (): boolean {
-        return (this._elapsed >= this._duration);
+    override isDone (): boolean {
+        return this._elapsed >= this._duration && !this.isUnknownDuration();
```

Comment:
If an action 's duration is unknown, we should not mark it as DONE until it isn't an `unknown duration` action.